packages/store/test/unit/data-provider/backend-system.test.ts

```diff
@@ -16,7 +16,7 @@ describe('Backend system data provider', () => {
 
     const logger = new ToolsLogger({ transports: [new NullTransport()] });
     beforeEach(() => {
-        jest.resetAllMocks();
+        // jest.resetAllMocks();
```

Comment:
Delete the line instead?

Reply:
Done in 3d67970ea55934680e91cd61e1b191bd65053b33